<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lwStudents">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lwScore">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStudent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leScore">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить студента</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddAfter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить студента перед выделенным студентом</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClear">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Редактировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSort">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Отсортировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCancel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDelete">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
